docs/Readme.md

```diff
@@ -0,0 +1,131 @@
+<p align="center"><img src="./img/lo.svg" width="260"></p>
+<p align="center">
+
+
+# Logging Operator Documentation
+
+### Welcome to the Logging Operator documentation!
+
+There are a lot of things a user of Logging Operator might need to know about. To help you manage this logging information, we've divided the operator docs into several sections.
+
+
+
+Logging-operator is a core part of the [Pipeline](https://beta.banzaicloud.io) platform, a Cloud Native application and devops platform that natively supports multi- and hybrid-cloud deployments with multiple authentication backends. Check out the developer beta:
+ <p align="center">
+   <a href="https://beta.banzaicloud.io">
+   <img src="https://camo.githubusercontent.com/a487fb3128bcd1ef9fc1bf97ead8d6d6a442049a/68747470733a2f2f62616e7a6169636c6f75642e636f6d2f696d672f7472795f706970656c696e655f627574746f6e2e737667">
+   </a>
+ </p>
+
+
+## Contents
+- **[Installation](./deploy/README.md)**
+  - [Deploy with Helm](./deploy/README.md#deploy-logging-operator-with-helm)
+  - [Deploy with Kubernetes Manifests](./deploy/README.md#deploy-logging-operator-from-kubernetes-manifests)
+- **[Supported Plugins](./plugins/Readme.md)**
+- **[Custom Resource Definitions](./crds.md)**
+- **[Troubleshooting](./troubleshooting.md)**
+- **[Fluentbit Configuration](./fluentbit.md)**
+- **[For Developers](./developers.md)**
+- **[Monitoring](./logging-operator-monitoring.md)**
+- **[Security](./security/README.md)**
+- **[Examples](./docs)**
+  - [Amazon S3 Output](examples/example-s3.md)
+  - [Amazon CloudWatch Output](examples/cloudwatch-nginx.md)
+  - [Elasticsearch Output](examples/es-nginx.md)
+  - [Grafana Loki Output](examples/loki-nginx.md)
+  - [Kafka Output](examples/kafka-nginx.md)
+  - [And more...](./examples)
+- **[Scaling](#scaling)**
+- **[Requirements](#requirements)**
+- **[Contributing](../README.md#contributing)**
+- **[Blogs](#blogs)**
+- **[Licence](#License)**
+---
+
+## Scaling
+In high scaling infrastructure the monitoring/logging components are also under huge pressure. The typical sign of this overload is when the fluentd can not handle its buffer directory size growth for a significant time. In this case, we can scaleup the [component.](./crds.md#Scaling). 
+
```

Comment:
Also please add the following:

_Note:_ One caveat when scaling fluentd is that when there are more then one instances sending logs to the same output, then the output will more likely receive chunks of messages out of order. Some outputs tolerate this (e.g. Elasticsearch) some do not, or requires fine tuning (e.g. Loki).